docs/content/en/docs/getting-started/optional/registrymirror.md

```diff
@@ -50,7 +50,7 @@ spec:
 * __Example__: ```port: 443```
 
 ### __ociNamespaces__ (optional)
-* __Description__: mapping from upstream registries to the locations of the private registry. When specified, the artifacts pulled from an upstream registry will be put in its corresponding location/namespace in the private registry. The target location/namespace must be already existing.
+* __Description__: when you need to mirror multiple registries, you can map the upstream registries to "namespaces" of the mirror.
```

Comment:
Is this a one-to-one mapping of a registry to a namespace? If so, I'd suggest wording like:
```suggestion
* __Description__: when you need to mirror multiple registries, you can map each upstream registry to the "namespace" of its mirror.
```
Let me know if I misunderstand you.

Reply:
sounds better to me, thanks!